### DISCUSSION TITLE
Layout Consistency

### DISCUSSION THREAD

**Discussion by MitchellGriggsTAFE** [MitchellGriggsTAFE/TafeCalc24#4] [Ideas]
To ensure consistency and user friendliness, I was thinking it would be a good idea to standardise the layout across the calculators. How do these suggestions sound?

- Light theme with a light background 
- Titles in bold with font size 28
- Headings in regular styling with font size 24
- Plain text in regular styling with font size 16
- Horizontally centred layout for all UI elements

**Comment by TravisGriggsProjects** [comment]
Sounds great, I'll make sure to include those settings in my code/layout.
